pudb/remote.py

```diff
@@ -84,6 +84,30 @@
 """
 
 
+class TelnetCharacters:
+    """Collection of characters from the telnet protocol RFC 854
+
+    This format for the telnet characters was adapted from the telnetlib module
+    which was removed from the C Python standard library in version 3.13. Only
+    the characters needed by pudb have been copied here. Additional characters
+    can be found by looking in the telnetlib code in Python 3.12 or in the
+    telnet RFC.
+
+    .. note::
+
+        This class is not intended to be instantiated.
+    """
+    # Telnet protocol characters
+    IAC = bytes([255])  # "Interpret As Command"
```

Comment:
IMHO, precompiled notation can be used:
```python
IAC = b'\xff'
```

Reply:
Okay, I pushed a change for that. I had just copied the format from cpython [here](https://github.com/python/cpython/blob/c0de843c12f4f3959799eaa730cd56852f6c20ef/Lib/telnetlib.py#L52-L57).